app/config.py

```diff
@@ -1,5 +1,15 @@
 import os
 
+def int_env(key, default):
+    """ Handle empty values for environment variables
+    and return the value as an int
+    """
+    val = os.environ.get(key, default)
+    if val == default or val == '':
```

Comment:
Can this be `if val == default or not val:` ?

Reply:
It could be, I thought this was a little more explicit. I will change it.